docs/InstallAdvEnv.md

```diff
@@ -0,0 +1,27 @@
+# Advanced Environments
+
+To use below features, each prerequisites are necessary.  
+The configure.sh also shows missed package to build configured as show below:
+```bash
+os$ ./tools/configure.sh artik053/grpc
+  Copy build environment files
+ 
+  CAUTION!! To build artik053/grpc, protoc should be installed
+  Find details at https://github.com/Samsung/TizenRT/blob/master/external/protobuf/README.md
+ 
+  Configuration is Done!
```

Comment:
In this case, protoc is not installed yet.
So is it reasonable to say that configuration is done?

Reply:
Regardless existance of packages, there is no problem on configuration.
Even if he installs missed packages, he does not need to re-configure because it affects only build.
That's why it says "configuration is done".

The previous step of build is this configuration so that this time we check and say missed packages. But actually it is not a step of configuration.